parser/prism/Translator.cc

```diff
@@ -95,7 +95,25 @@ ast::ExpressionPtr mergeStrings(core::MutableContext ctx, core::LocOffsets loc,
     }
 }
 
-// Member function implementation for desugarDString
+// Extract the content and location of a Symbol node.
+// This is handy for `desugarSymbolProc`, where it saves us from needing to dig and
+// cast to extract this info out of an `ast::Literal`.
+pair<core::NameRef, core::LocOffsets> Translator::translateSymbol(pm_symbol_node *symbol) {
+    auto location = translateLoc(symbol->base.location);
+
+    auto unescaped = &symbol->unescaped;
+    // TODO: can these have different encodings?
+    auto content = ctx.state.enterNameUTF8(parser.extractString(unescaped));
```

Comment:
This comment is repeated in a bunch of places, I'll follow up to clean them all up at the same time.

The answer is basically "no", Sorbet only supports unicode (perhaps only utf8, specifically).